extensions/script/python/ExecutePythonProcessor.cpp

```diff
@@ -35,155 +35,184 @@ namespace python {
 namespace processors {
 
 core::Property ExecutePythonProcessor::ScriptFile("Script File",  // NOLINT
-    R"(Path to script file to execute)", "");
+    R"(Path to script file to execute. Only one of Script File or Script Body may be used)", "");
+core::Property ExecutePythonProcessor::ScriptBody("Script Body",  // NOLINT
```

Comment:
Using property builder (withdescription, blah blah) can help breaking these lines without linter errors. And improve readability as well. 
You can also specify some to be required.

Reply:
Replaced.